src/m3u8/m3u8-parser.js

```diff
@@ -436,7 +440,8 @@
               // setup an encryption key for upcoming segments
               key = {
                 method: entry.attributes.METHOD || 'AES-128',
-                uri: entry.attributes.URI
+                uri: entry.attributes.URI,
+                iv: entry.attributes.IV || null
```

Comment:
entry.attributes.IV is there and was working, other than the whole 0x prefix thing, but it needs to propagate to the decryptor somehow.